### PR TITLE
web > status: Fix pluralization of client socket connections

### PR DESCRIPTION
**Pull request by bdd**
Instead of changing `> 1` to `== 1`, use `yes` variable in consistent with the rest of the file.

Fixes:
- `Yes, 1 connections` to `Yes, 1 connection`.
- `Yes, 2 connection` to `Yes, 2 connections`.

### CODE REVIEW

File: src/web.cpp [hunk 1]
```diff
@@ -752,17 +752,16 @@ void handleRoot() {
...
         if (ConfigSettings.connectedClients > 0) {
-            if (ConfigSettings.connectedClients > 1) {
-                doc[connectedSocketStatus] = "Yes, " + String(ConfigSettings.connectedClients) + "connection";
-                doc[connectedSocket] = readableTime;
-            } else {
-                doc[connectedSocketStatus] = "Yes, " + String(ConfigSettings.connectedClients) + " connections";
-                doc[connectedSocket] = readableTime;
-            }
+            char str[32];
+            int n = ConfigSettings.connectedClients;
```
Comment (Tarik2142) [inline src/web.cpp:756]: I think it is better to make the variable "n" a constant. The int type is redundant, uint8_t is better

Comment (Tarik2142) [reply src/web.cpp:756]: > Do you mean to suggest I make this const int

`const uint8_t`

> I can go change the connectedClients in config.h from int to uint8_t to save a few bytes of memory

Yes, that would be great.

> Looking at the emitted assembly, it uses 32-bit general purpose registers, anyway.

That's strange, I always thought that specifying a var type should work in the arduino framework

**Comment by Tarik2142** [comment]
Renaming Version.h -> version.h causes a conflict, so I think it's better to leave Version.h

**Comment by bdd** [comment]
> Renaming Version.h -> version.h causes a conflict, so I think it's better to leave Version.h

Ah. That wasn't intended to slip into this one. There's another PR for that.
Fixed and force pushed.

**Comment by Tarik2142** [comment]
Thank you for your contribution to the development of SLZB-06

**Pull request merged**